mongo/description/server.go

```diff
@@ -31,35 +31,37 @@ type SelectedServer struct {
 type Server struct {
 	Addr address.Address
 
-	Arbiters              []string
-	AverageRTT            time.Duration
-	AverageRTTSet         bool
-	Compression           []string // compression methods returned by server
-	CanonicalAddr         address.Address
-	ElectionID            primitive.ObjectID
-	HeartbeatInterval     time.Duration
-	HelloOK               bool
-	Hosts                 []string
-	IsCryptd              bool
-	LastError             error
-	LastUpdateTime        time.Time
-	LastWriteTime         time.Time
-	MaxBatchCount         uint32
-	MaxDocumentSize       uint32
-	MaxMessageSize        uint32
-	Members               []address.Address
-	Passives              []string
-	Passive               bool
-	Primary               address.Address
-	ReadOnly              bool
-	ServiceID             *primitive.ObjectID // Only set for servers that are deployed behind a load balancer.
-	SessionTimeoutMinutes uint32
-	SetName               string
-	SetVersion            uint32
-	Tags                  tag.Set
-	TopologyVersion       *TopologyVersion
-	Kind                  ServerKind
-	WireVersion           *VersionRange
+	Arbiters          []string
+	AverageRTT        time.Duration
+	AverageRTTSet     bool
+	Compression       []string // compression methods returned by server
+	CanonicalAddr     address.Address
+	ElectionID        primitive.ObjectID
+	HeartbeatInterval time.Duration
+	HelloOK           bool
+	Hosts             []string
+	IsCryptd          bool
+	LastError         error
+	LastUpdateTime    time.Time
+	LastWriteTime     time.Time
+	MaxBatchCount     uint32
+	MaxDocumentSize   uint32
+	MaxMessageSize    uint32
+	Members           []address.Address
+	Passives          []string
+	Passive           bool
+	Primary           address.Address
+	ReadOnly          bool
+	ServiceID         *primitive.ObjectID // Only set for servers that are deployed behind a load balancer.
+	// Deprecated: Use SessionTimeoutMinutesPtr instead.
+	SessionTimeoutMinutes    uint32
+	SessionTimeoutMinutesPtr *uint32
```

Comment:
This change seems fine, but there will be some amount of conversion either way. Updating to int64, there will be cases where we need to convert int64 to uint32 and vice versa. For example, using the "sessionTimeoutMinutes" variable in opsmg_deployment.go

`localLogicalSessionTimeoutMinutes` is not production safe, per the [documentation](https://www.mongodb.com/docs/manual/reference/parameters/#mongodb-parameter-param.localLogicalSessionTimeoutMinutes).  Part of the goal for the ticket is to interpret session support from the server via the existence of this timeout. I'm not sure the server could ever return a negative value for "logicalSessionTimeoutMinutes" but even if it could I would assume a negative value should be treated the same as a zero value, i.e. no timeout.